tests/functional/pages/surveys/household_composition/summary.page.js

```diff
@@ -3,17 +3,47 @@ import SummaryPage from '../../summary.page'
 class HouseholdCompositionSummary extends SummaryPage {
 
   clickEdit() {
-    browser.click('[data-qa="household-full-name-edit"]')
+    browser.click('[data-qa="first-name-edit"]')
     return this
   }
 
-  isNameDisplayed(name) {
-    return browser.element('[data-qa="household-full-name-answer"]').getText() === name
+  isNameDisplayed(index, first = '', middle = '', last = '') {
```

Comment:
This method has got pretty complex, in the test we could do 
```
expect(HouseholdCompositionSummary.getFirstName(0)).to.equal('Joe')
expect(HouseholdCompositionSummary.getSurname(0)).to.equal('Bloggs')
```
The benefits being we will know which part is failing the first name, middle name and surname part. Also we will know what was actually output instead of the name